src/dataProcessor/DataProcessor.java

```diff
@@ -0,0 +1,130 @@
+package dataProcessor;
+
+import graphs.Graph;
+
+public class DataProcessor {       //TODO: when implemented differentiate between directed and undirected,weighted and unweighted edges
+                                    //TODO implement writing output out of given graph
+    public enum Type{
+        adjacencyMatrix, edgeList,incidenceMatrix;  //TODO: implement adjacencyList
```

Comment:
As far as I know enums should be all Upper case and underscore.
ADJACENCY_MATRIX
EDGE_LIST etc.